packages/osl-operator-bundle-image/generated/bundle/manifests/logic-operator-rhel8-builder-config_v1_configmap.yaml

```diff
@@ -11,7 +11,7 @@ data:
     under the License is distributed on an\n# \"AS IS\" BASIS, WITHOUT WARRANTIES
     OR CONDITIONS OF ANY\n# KIND, either express or implied.  See the License for
     the\n# specific language governing permissions and limitations\n# under the License.\n\nFROM
-    docker.io/apache/incubator-kie-sonataflow-builder:main AS builder\n\n# variables
+    docker.io/apache/incubator-kie-sonataflow-builder:1.36.0 AS builder\n\n# variables
```

Comment:
This image name is also wrong, we should use the ones from prod. I'll take a look tomorrow, but you can change all these images to the ones from prod manually.